src/components/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,49 @@
+import { Loader } from '../Loader';
+import { PeopleTable } from '../Loader/PeopleTable';
+import { getPeople } from '../services/api';
+import { useEffect, useState } from 'react';
+import { Person } from '../../types';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    getPeople()
+      .then(data => {
+        const dataUpdated = data.map(p => {
+          return {
+            ...p,
+            mother: data.find(person => person.name === p.motherName),
+            father: data.find(person => person.name === p.fatherName),
+          };
```

Comment:
This code has a performance issue. In the map function, you are using the find function which has a time complexity of O(n). Since this is inside a map function which also has a time complexity of O(n), the overall time complexity becomes O(n^2). This can slow down your application if you have a large amount of data. I would suggest creating an object that maps person names to their data before the map function, and then use this object inside the map function to find the mother and father. This will reduce the time complexity to O(n).